edisgo/tools/tools.py

```diff
@@ -219,39 +332,76 @@ def select_cable(edisgo_obj, level, apparent_power):
         Number of necessary parallel cables.
 
     """
-
-    cable_count = 1
-
+    if not cos_phi:
+        cos_phi = 0.95
```

Comment:
This is not necessary because 0.95 is already the default value in the function head.